tests/test_dependency_rendering.py

```diff
@@ -471,29 +440,18 @@ def test_multiple_components_dependencies(self):
             count=1,
         )
 
-        # We expect to find this:
-        # ```js
-        # Components.manager._loadComponentScripts({
-        #     loadedCssUrls: [&quot;/components/cache/OtherComponent_6329ae.css/&quot;, &quot;/components/cache/SimpleComponentNested_f02d32.css/&quot;, &quot;style.css&quot;, &quot;style2.css&quot;, &quot;xyz1.css&quot;],
-        #     loadedJsUrls: [&quot;/components/cache/OtherComponent_6329ae.js/&quot;, &quot;/components/cache/SimpleComponentNested_f02d32.js/&quot;, &quot;script.js&quot;, &quot;script2.js&quot;, &quot;xyz1.js&quot;],
-        #     toLoadCssTags: [],
-        #     toLoadJsTags: [],
-        # });
-        # ```
-        self.assertEqual(
-            rendered.count(
-                "loadedJsUrls: [&quot;/components/cache/OtherComponent_6329ae.js/&quot;, &quot;/components/cache/SimpleComponentNested_f02d32.js/&quot;, &quot;script.js&quot;, &quot;script2.js&quot;, &quot;xyz1.js&quot;],"
-            ),
-            1,
-        )
-        self.assertEqual(
-            rendered.count(
-                "loadedCssUrls: [&quot;/components/cache/OtherComponent_6329ae.css/&quot;, &quot;/components/cache/SimpleComponentNested_f02d32.css/&quot;, &quot;style.css&quot;, &quot;style2.css&quot;, &quot;xyz1.css&quot;],"
-            ),
-            1,
+        self.assertInHTML(
+            """
+            <script type="application/json" data-djc>
+                {"loadedCssUrls": ["/components/cache/OtherComponent_6329ae.css/", "/components/cache/SimpleComponentNested_f02d32.css/", "style.css", "style2.css", "xyz1.css"],
```

Comment:
That slash after the `OtherComponent_6329ae.css` isn't too nice. Could we remove it?

Reply:
Updated :)